core/src/main/java/com/radixdlt/RunningRadixNode.java

```diff
@@ -96,8 +96,6 @@ private RunningRadixNode(Injector injector) {
   }
 
   public static RunningRadixNode run(UnstartedRadixNode unstartedRadixNode) {
-    log.info("Starting Radix node");
```

Comment:
Out of interest, why was this removed?

Reply:
too many open branches :sweat_smile: I've replaced this with `https://github.com/radixdlt/babylon-node/pull/548/files#diff-d14e0db8dc21558566c417273e02c97a19d29b5502fd22260c256bdbb412dbb6R178` in a different PR